### ISSUE TITLE
Monika Nickname

### ISSUE THREAD

**Issue by LeStyleau** [Monika-After-Story/MonikaModDev#10289]
Monika don't have the nickname I've given her, and I can't give her a new nickname.
How I can fix that?

**Comment by Booplicate** [comment]
Did you use a bad nickname?

**Comment by LeStyleau** [comment]
I don't think so, I close the game with her having the nickname I give her and when I reopen it her nickname was back to "Monika" and the option to change her nickname disappear.

**Comment by Booplicate** [comment]
If you did something mean, that could also affect it. Make sure there's nothing to apologise for, do something nice, and see in a few days if the option is back.
